PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
It's an iterable of length 2. There are places in calling code (i.e. django) where a 2 element list is passed in. 

We've hit this usage before when making things tighter with the type/size checking.